src/xo-app/sr/tab-disks.js

```diff
@@ -89,13 +51,60 @@ const COLUMNS = [
     itemRenderer: vdi => formatSize(vdi.size),
     sortCriteria: vdi => vdi.size,
   },
+  {
+    name: _('vdiVms'),
+    component: connectStore(() => {
+      const getVbds = createGetObjectsOfType('VBD').pick(
+        (_, props) => props.item.$VBDs
+      )
+      const getVmIds = createSelector(getVbds, vbds => map(vbds, 'VM'))
+      const getVms = createGetObjectsOfType('VM').pick(getVmIds)
```

Comment:
Actually, we're also listing `VDI-snapshot`s here so you'll also need to get the `VM-snapshot`s and to handle the `Link`s properly. My bad. :innocent: